audb/core/load.py

```diff
@@ -1067,7 +1062,7 @@ def load_media(
 
             # load missing media
             if not db_is_complete:
-                _load_media(
+                cached_versions = _load_media(
```

Comment:
We are not re-using `cached_versions` in `load_media()`, so we don't have to assign it here, but can stay with

```suggestion
                _load_media(
```

In addition, i would propose to move the line
```python
cached_versions = None
```
from the beginning of this function to the line before we call `_load_media()` as it is only needed here.

Reply:
I left it there because I'm not sure yet how we will handle the blocking of the databases and we may need access to the cached versions to release the lock. But I'm also fine with removing it now and adding it later if needed. Most likely, we will anyway handle that inside the function in a `with` statement (we will probably introduce a helper class that can lock several folders at once).

Reply:
I would be in favor of first removeing it here, and re-adding it later if needed.

Reply:
I removed the `cached_files = None` line and directly pass `None` to `_load_media()`. The return value is also not stored anymore.